### ISSUE TITLE
Access Thimble with Firefox Accounts

### ISSUE THREAD

**Issue by ranjithrajvasam** [mozilla/thimble.mozilla.org#2440]
Most of the Mozilla services are accessible by Firefox accounts, that is really good thing to appreciate.
Thimble being a Mozilla service, it's good to have a feature to login via Firefox Accounts.
Creating a separate account and managing the password is little irritating.  

Ref: https://support.mozilla.org/en-US/kb/access-mozilla-services-firefox-accounts
Tags: Feature Request, High Priority

**Comment by hkirat** [comment]
A discussion regarding shifting to social sign in going on [here](https://github.com/mozilla/thimble.mozilla.org/issues/2431)

**Comment by ryanwarsaw** [comment]
I'd love to support Firefox accounts in the future, but currently we just aren't at a point where this is easily do-able. Adding strategies takes a lot of time and resources, we'd need to make a new `passport` strategy for Firefox accounts, and then it'd need to be reviewed by DevOps for security.

That assumes we've got a completed implementation in place, which is certainly not the case. Currently it's just a one person team (myself) and if I had to predict when we would be in a good place to start considering additional strategies, I'd say around beginning of November. 

I'll leave it up to @gideonthomas if we want to keep this open and in consideration until then, or re-visit once we're much closer to completion on work regarding this front.
